event_session/models/event_mail.py

```diff
@@ -50,30 +50,21 @@ def _compute_done(self):
                     and all(line.mail_sent for line in event_mail.mail_registration_ids)
                 )
 
-    @api.depends(
-        "event_id.state",
-        "event_id.date_begin",
-        "interval_type",
-        "interval_unit",
-        "interval_nbr",
-    )
+    @api.depends()
```

Comment:
Why removing dependencies here?

Reply:
Same dependencies than super I think.
But I do think these are missing:

- `session_id.date_begin`
- `session_id.date_end`

Reply:
Thanks @ivantodorovich you're right :+1: